<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_bearb_faufruf</class>
 <widget class="QDialog" name="Dialog_bearb_faufruf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>471</width>
     <height>491</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_z">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_x">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QPushButton" name="pushButton_wkzwahl">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Bezug</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Werkzeugnummer</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Pos Y</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_afb"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Ausf√ºhrbedingung</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Abfahrtyp</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Eintauch-Tiefe</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_kor"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Pos Z</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_bezug">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Anfahrweg</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Anfahrtyp</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_ti"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Pos X</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Korrektur</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="lineEdit_y">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="lineEdit_wkz"/>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Abfahrweg</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLineEdit" name="lineEdit_anweg"/>
    </item>
    <item row="12" column="1">
     <widget class="QLineEdit" name="lineEdit_abweg"/>
    </item>
    <item row="9" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_antyp"/>
    </item>
    <item row="10" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_abtyp"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>540</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btn_ok">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_abbrechen">
      <property name="text">
       <string>Abbrechen</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_x</tabstop>
  <tabstop>lineEdit_y</tabstop>
  <tabstop>lineEdit_z</tabstop>
  <tabstop>lineEdit_ti</tabstop>
  <tabstop>comboBox_kor</tabstop>
  <tabstop>comboBox_bezug</tabstop>
  <tabstop>lineEdit_afb</tabstop>
  <tabstop>lineEdit_wkz</tabstop>
  <tabstop>comboBox_antyp</tabstop>
  <tabstop>comboBox_abtyp</tabstop>
  <tabstop>lineEdit_anweg</tabstop>
  <tabstop>lineEdit_abweg</tabstop>
  <tabstop>btn_ok</tabstop>
  <tabstop>btn_abbrechen</tabstop>
  <tabstop>pushButton_wkzwahl</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
